src/applications/personalization/dashboard/components/benefit-payments-and-debts/BenefitPaymentsAndDebt.jsx

```diff
@@ -83,27 +89,33 @@ const BenefitPaymentsAndDebt = ({
               icon="dollar-sign"
               newTab
               text="Manage your direct deposit"
+              /* eslint-disable react/jsx-no-bind */

```

Comment:
It's better to use `/* eslint-disable-next-line react/jsx-no-bind */` instead of disabling and enabling eslint.